M:tG Life/UI/ZPLColorSelectorView.m

```diff
@@ -40,6 +40,52 @@ typedef NS_ENUM(NSUInteger, ZPLCardinalColor) {
     }
 }
 
+UIColor* cardinalColorToUIColor(ZPLCardinalColor cardinalColor) {
```

Comment:
We'll almost certainly need this stuff in other views. You don't have to move it now but keep move it next time you need it.